common/src/test/scala/com/gu/salesforce/Fixtures.scala

```diff
@@ -5,17 +5,19 @@ object Fixtures {
   val salesforceId = "003g000001UnFItAAN"
   val email = "6cwdm8aler7z9r6nwbc@gu.com"
   val name = "6cWdM8AlER7z9R6nWBc"
+  val dummyValue = "Dummy Value"
   val allowMail = false
   val upsertJson =
     s"""{
       "newContact": {
         "IdentityID__c": "$idId",
-         "Email": "$email",
-         "FirstName": "$name",
-         "LastName": "$name",
-         "Allow_Membership_Mail__c": $allowMail,
-         "Allow_3rd_Party_Mail__c": $allowMail,
-         "Allow_Guardian_Related_Mail__c": $allowMail
+        "Email": "$email",
+        "FirstName": "$name",
+        "LastName": "$name",
+        "MailingCountry": "$dummyValue",
```

Comment:
Why does this include country but not state?

Reply:
Because if the state is missing as it will be in the uk we would send nothing and this is what this tests. As @JustinPinner suggested I'll add another test for the case where we do have a state.